xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -1156,9 +1156,8 @@ void DX::DeviceResources::CheckDXVA2SharedDecoderSurfaces()
 
   m_DXVA2SharedDecoderSurfaces =
       ad.VendorId == PCIV_Intel ||
-      (ad.VendorId == PCIV_NVIDIA && driver.valid && driver.majorVersion >= 465) ||
```

Comment:
Why remove NVIDIA driver condition?

Nvidia driver > 465 supports true shared path without any other condition i.e Windows 10 < Creators Update and without this PR (fence).

Also `D3D_FEATURE_LEVEL_12_1` is relevant only for AMD because identifies HW gen that supports true shared path.

Please change to already commented:

```
  m_DXVA2SharedDecoderSurfaces =
      ad.VendorId == PCIV_Intel ||
      (ad.VendorId == PCIV_NVIDIA && driver.valid && driver.majorVersion >= 465) ||
      (ad.VendorId == PCIV_AMD && driver.valid && driver.majorVersion >= 30 &&
       m_d3dFeatureLevel >= D3D_FEATURE_LEVEL_12_1) ||
      CSysInfo::IsWindowsVersionAtLeast(CSysInfo::WindowsVersionWin10_1703));
```

Anyway this change should go in with a separate commit or follow-up PR because we must first confirm that it works for Nvidia and AMD. Maybe it only works for Nvidia and then it will have to be modified.

Reply:
This is for testing purposes with a special build at this point, so things are wide open to activate for a large number of systems on purpose.
The merge button is not on fire and I had no intention to merge as is.
I'm publishing a couple test builds in the forum to receive feedback on what works and what doesn't.
We can then discuss and agree on the conditions before a merge.

Reply:
... but I didn't account for the fact that maybe nVidia and/or AMD had a working shared path under Windows 8.
Did they?

Reply:
Yes, Nvidia with drivers >= 465 should work since no one has complained.

AMD new HW (which is reported with DirectX FL 12.1) should work but since Windows 8 is limited to FL 11.1, this path is not used.

Reply:
ok so let's not open that door, unless someone shows up with a computer that allows testing.